### DISCUSSION TITLE
Question on updating to FreshRSS-edge "beta" version

### DISCUSSION THREAD

**Discussion by bigl00z3** [FreshRSS/FreshRSS#5124] [Q&A] [answered]
Quick question as I haven't found the answer in the discussion list. 
I want to try the latest Edge version. Is it possible to edit `config.default.php` to call for update of the unstable release when selecting `Check for new updates` ?
![image](https://user-images.githubusercontent.com/8886922/220173208-c419e475-9567-4439-8932-c0bfd91d6a4b.png)

Or is the only way to install the update by uploading the files manually (FTP to my shared host)?

Nico

**Comment by aledeg** [comment, marked as answer]
The update system works only with the tagged versions from the update server. If you want to use the latest edge version, you need to use either FTP or git to get it.
At the moment, there is no alternative to get the code.

There are plans to change this behavior but it requires a lot of work. If you feel like it, you might give a hand to get it done quicker.

**Reply by bigl00z3** [reply]
Thank you for your reply. I will go with the FTP method.
` If you feel like it, you might give a hand to get it done quicker.`
I doubt my programing skills would help
